packages/matplotlib/test_matplotlib.py

```diff
@@ -2,15 +2,15 @@ def test_matplotlib(selenium_standalone):
     selenium = selenium_standalone
     selenium.load_package("matplotlib")
     selenium.run("from matplotlib import pyplot as plt")
-    selenium.run("plt.figure()")
-    selenium.run("plt.plot([1,2,3])")
+    selenium.run("plt.figure(); pass")
+    selenium.run("x = plt.plot([1,2,3])")
```

Comment:
@hoodmane can you please shortly explain this change?

Reply:
Well I'm not exactly sure why this is needed, but the tests now fail without it. My belief is that it is a problem caused when `selenium` tries to serialize a `PyProxy`, it somehow is raising an exception. I'm not really sure why it worked before because it seems to me unlikely that the return value of `plt.figure()` was a dictionary before?

Anyways the point of this test was to check that `plt.figure()` succeeded and not whether `selenium` knew how to serialize it. Probably it would be better to fix the serialization, but that seems like an independent change.

Reply:
Saying `x = plt.plot([1,2,3])` also blocks the serialization of the plot. I blocked it by assigning the plot to `x` because most of the other similar tests also say `x = plt.plot([1,2,3])`.

Reply:
@hoodmane Ok, thanks for the explanation.